clean.sh

```
@@ -0,0 +1,27 @@
+# 1. Tuer tous les processus liés à Nexus ou au conteneur
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)